pkg/exec/cmd_run_result.go

```diff
@@ -62,3 +62,11 @@ func (r CmdRunResult) WithFriendlyYAMLStrings() CmdRunResult {
 		Finished: r.Finished,
 	}
 }
+
+// IsEmpty returns true if CmdRunResult is empty, otherwise returns false
```

Comment:
nit: Could this be turned into something like the following? And I think the comment is somewhat redundant, the function is so small and just does one thing

```
func (r CmdRunResult) IsEmpty() bool {
  return r == (CmdRunResult{})
}
```

Reply:
agree with the implementation note;  i think the comment is b/c it's a public method and we have that blanket policy turned on.  i do think we can bump the SHA in the linter config rather than getting trapped in absurd/uninformative comments but imo something like `IsEmpty tells whether the receiver is equal to its zero value` is maybe a tiny bit more helpful?


I think  `empty` is overloaded because it can mean "has no fields"  but also maybe "is equivalent to what you'd get with a `{}` zero-value / default-constructor",  right?

Reply:
> nit: Could this be turned into something like the following?

Absolutely. Thanks for the suggestion :)

> And I think the comment is somewhat redundant, the function is so small and just does one thing

Not a big fan of redundant comments myself, as @joe-kimmel-vmw mentioned I had to do it because of the blanket policy turned on. But if y'all agree that it is not needed, I can try to bump the SHA in linter config.